_aggregations/bucket/sampler.md

```diff
@@ -8,7 +8,7 @@ nav_order: 170
 
 # Sampler aggregations
 
-If you're aggregating over millions of documents, you can use a `sampler` aggregation to reduce its scope to a small sample of documents for a faster response. The `sampler` aggregation selects the samples by top-scoring documents.
+If you're aggregating a very large number of documents, you can use a `sampler` aggregation to reduce its scope to a small sample of documents for a faster response. The `sampler` aggregation selects the samples by top-scoring documents.
```

Comment:
What constitutes a "very large number"? This is subjective. We need to give user an idea of what is means for them.

Reply:
"over millions" is also kind of subjective. As a user I'm not sure if this means > 1 million or >10 million or > 100 million. But more importantly there isn't really a threshold here where above x documents the sampler aggregation is useful and below x documents it isn't.

To better illustrate this, there's really 2 (main) use cases for this aggregation type:
1. Some aggregations, like `significant_terms`, still produce relevant results on just a sample of the data so in those cases the sampler aggregation can be used to reduce the cost of running the query.
2. In cases where the query may have a lot of low-quality matches and a few high-quality matches, the sampler aggregation can help focus on just the high-quality matches.